manifests/s/SamyGO/PVRDecoder/1.0/SamyGO.PVRDecoder.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.PVRDecoder

```

```suggestion
PackageIdentifier: SamyGo.PVRDecoder
```
Must match existing publisher casing or there will be Git conflicts

---

Please update file name and path to match this change.